tycho-common/src/models/blockchain.rs

```diff
@@ -341,6 +342,31 @@ pub enum BlockTag {
     /// Block by number
     Number(u64),
 }
+#[derive(Debug, Clone)]
+/// An entry point to trace. Different types of entry points tracing will be supported in the
+/// future. Like RPC debug tracing, symbolic execution, etc.
+pub enum EntryPoint {
```

Comment:
Can you please add this:

https://doc.rust-lang.org/reference/attributes/type_system.html#r-attributes.type-system.non_exhaustive